lib/vquic/ngtcp2.c

```diff
@@ -885,15 +886,17 @@ static void qs_disconnect(struct quicsocket *qs)
   char buffer[NGTCP2_MAX_UDP_PAYLOAD_SIZE];
   ngtcp2_tstamp ts;
   ngtcp2_ssize rc;
+  ngtcp2_connection_close_error errorcode;
+
   if(!qs->conn) /* already closed */
     return;
+  memset(&errorcode, 0, sizeof(errorcode));
+  errorcode.type = NGTCP2_CONNECTION_CLOSE_ERROR_CODE_TYPE_APPLICATION;
```

Comment:
would be nice to use this if application error type is needed:

```c
ngtcp2_connection_close_error_set_application_error(&errorcode, NGHTTP3_H3_NO_ERROR, NULL, 0);
```

Reply:
Ah, nice. BTW, does this function also clear the other fields of the `ngtcp2_connection_close_error` struct or do I need to keep the memset?

Reply:
And man, do you make **long** function names...

Reply:
The function initializes all fields.  No memset is required here.
And long functions names, it is a known issue.  I think application API tends to be longer because it does rather complex stuff and distinguish out of similar functions so that makes them explicit.  But I probably review these names before ngtcp2 v1.0 to see whether I can shorten these names.

Reply:
Thanks!